runtime/setup.py

```diff
@@ -169,16 +170,14 @@ def find_git_submodule_revision(submodule_path):
 
 
 try:
-    version_info = load_version_info()
+    version_info = load_version_info(VERSION_INFO_RC_FILE)
 except FileNotFoundError:
-    print("version_info.json not found. Using defaults", file=sys.stderr)
-    version_info = {}
+    print("version_info_rc.json not found. Default to dev build")
+    version_info = load_version_info(VERSION_INFO_FILE)
 git_versions = find_git_versions()
 
 PACKAGE_SUFFIX = version_info.get("package-suffix") or ""
 PACKAGE_VERSION = version_info.get("package-version")
-if not PACKAGE_VERSION:
-    PACKAGE_VERSION = f"0.dev0+{git_versions.get('IREE') or '0'}"
```

Comment:
Could we keep this git version tagging for dev builds?

Current proposal:

Type of build | Version format | Version example
------------- | -------------- | ---------------
Stable release (PyPI) | `X.Y.Z` | `3.0.0`
Nightly release (GitHub `schedule`) | `X.Y.ZrcYYYYMMDD` | `3.0.1rc20241029`
Dev release (GitHub `pull_request`) | `X.Y.Z.devNN` | `3.0.1.dev50`
Local build | `X.Y.Z.dev` | `3.0.1.dev`

Alternate:

Type of build | Version format | Version example
------------- | -------------- | ---------------
Stable release (PyPI) | `X.Y.Z` | `3.0.0`
Nightly release (GitHub `schedule`) | `X.Y.ZrcYYYYMMDD` | `3.0.1rc20241029`
Dev release (GitHub `pull_request`) | `X.Y.Z.devNN` | `3.0.1.dev50`
Local build | `X.Y.Z.dev+SHA` | `3.0.1.dev+bbbc8189011b3dbbfb902771955eae39ec57de60`

Reply:
Fine for me :) Want to add this to the RFC?